features/jetpack-beta.php

```diff
@@ -36,7 +36,10 @@ function ( &$app = null, $features, $domain ) use ( $defaults ) {
 				$features = array_merge( $defaults, $features );
 				if ( $features['jetpack-beta'] ) {
 					debug( '%s: Adding Jetpack Beta Tester Plugin', $domain );
-					add_jetpack_beta_plugin();
+					add_jetpack_beta_plugin( false );
+				} elseif ( $features['jetpack-beta-dev'] ) {
```

Comment:
Should we flip this logic? If both are selected, for whatever reason, go with the dev version?